src/main/java/me/totalfreedom/totalfreedommod/command/Command_clearchat.java

```diff
@@ -0,0 +1,33 @@
+package me.totalfreedom.totalfreedommod.command;
+
+import me.totalfreedom.totalfreedommod.rank.Rank;
+import me.totalfreedom.totalfreedommod.util.FUtil;
+
+import org.bukkit.*;
+import org.bukkit.command.Command;
+import org.bukkit.command.CommandSender;
+import org.bukkit.entity.Player;
```

Comment:
these imports aren't needed if you're using the wildcard btw, so just remove the wildcard instead